packages/components/rollup.config.js

```diff
@@ -0,0 +1,53 @@
+import json from '@rollup/plugin-json';
+import babel from '@rollup/plugin-babel';
+import peerDepsExternal from 'rollup-plugin-peer-deps-external';
+import postcss from 'rollup-plugin-postcss';
+import resolve from '@rollup/plugin-node-resolve';
+import sassFunctions from 'sass-functions';
+import sassGlobImporter from 'node-sass-glob-importer';
+import del from 'rollup-plugin-delete';
+import path from 'path';
+import sass from 'sass';
+// eslint-disable-next-line import/extensions
+import pkg from './package.json';
+
+export default {
+  input: 'src/index.jsx',
+  output: [
+    {
+      file: pkg.module,
+      format: 'es',
+      sourcemap: true,
+    },
+    {
+      file: pkg.main,
+      format: 'cjs',
+      sourcemap: true,
+    },
+  ],
+  plugins: [
+    del({
+      targets: path.join(pkg.module, '../*'),
```

Comment:
what is this deleting exactly?

Reply:
this is cleaning up the lib directory, better use `__dirname` and `'lib/*'` ?

Reply:
```
      targets: path.join(__dirname, 'lib/*'),
```

Reply:
clearer yes